src/main/java/frc/robot/constants/ShooterConstants.java

```diff
@@ -6,4 +6,26 @@ public class ShooterConstants {
     public static final int BOTTOM_MOTOR_ID = 5;
     public static final int TOP_MOTOR_ID = 6;
 
+    // constants for Pids
+    public static final double TOP_P = 0.00005;
+    public static final double TOP_I = 0;
+    public static final double TOP_D = 0;
+
+    public static final double S = 0;
+    public static final double V = 1.0/6000;
+
+    public static final double BOTTOM_P = 0.00005;
+    public static final double BOTTOM_I = 0;
+    public static final double BOTTOM_D = 0;
```

Comment:
You probably don't need 2 sets of PID constants for identical sides of the shooter.

Reply:
Two sets are not needed and were removed, but two sets do not hurt.